autoload/lsp/handlers.vim

```diff
@@ -25,8 +25,9 @@ def ProcessInitializeReply(lspserver: dict<any>, req: dict<any>, reply: dict<any
   lspserver.caps = caps
 
   if opt.lspOptions.autoComplete && caps->has_key('completionProvider')
-    var triggers = caps.completionProvider.triggerCharacters
-    lspserver.completionTriggerChars = triggers
+    if caps.completionProvider->has_key('triggerCharacters')
+	lspserver.completionTriggerChars = caps.completionProvider.triggerCharacters
```

Comment:
Can you change this to?

lspserver.completionTriggerChars = caps.completionProvider->get('triggerCharacters', [])